scope.py

```diff
@@ -536,6 +536,7 @@ def train(
         from scope.utils import Dataset
 
         train_config = self.config["training"]["classes"][tag]
```

Comment:
Should this variable be called something else now?

Reply:
That's a good idea - I'll revise with a more accurate name.